tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js

```diff
@@ -0,0 +1,231 @@
+const { test, expect } = require( '@playwright/test' );
+const WpAdminPage = require( '../../../pages/wp-admin-page' );
+const EditorPage = require( '../../../pages/editor-page' );
+
+test.describe( 'Nested Tabs tests', () => {
+	test( 'Count the number of icons inside the Add Section element', async ( { page }, testInfo ) => {
```

Comment:
This is indicated as a new test. Once we compare the right version of 3.10.0 that shouldn't be the case anymore.

Reply:
o.k

Reply:
I have sent a fix. That was my error.